### DISCUSSION TITLE
Model Accuracy

### DISCUSSION THREAD

**Discussion by npiroozan** [deepmodeling/deepmd-kit#1832] [Q&A]
Hello,

I hope you are well!

I have a question regarding measurement of model accuracy.  There are several rmse values for trn, force, and energy.  

Which one is most important and is it possible to get values below 10^-2 for rmse_trn?

Thank you very much.

Warm regards

**Comment by njzjz** [comment]
The importance and the value of `rmse_trn` are decided by your parameter.

https://docs.deepmodeling.com/projects/deepmd/en/master/train/train-input.html#argument:loss[ener]/start_pref_e
https://docs.deepmodeling.com/projects/deepmd/en/master/train/train-input.html#argument:loss[ener]/limit_pref_e
https://docs.deepmodeling.com/projects/deepmd/en/master/train/train-input.html#argument:loss[ener]/start_pref_f
https://docs.deepmodeling.com/projects/deepmd/en/master/train/train-input.html#argument:loss[ener]/limit_pref_f

**Comment by wanghan-iapcm** [comment]
Other factors:
- The error in the training data defines the lower bound of the training error 
- The number of training steps
- In some cases you may want to try the hybridization of se_2_a and se_3

**Reply by npiroozan** [reply]
Thank you very much!
